datadog_checks_dev/datadog_checks/dev/tooling/jmx.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+JVM_E2E_METRICS = [
```

Comment:
Note: let's put that in `datadog_checks_dev`, so other integration can use that too.